### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by petersutter**
**How to categorize this PR?**

/area dev-productivity
/kind enhancement
/priority normal

**What this PR does / why we need it**:
Adapt the github pull request template to the new release note categories. 
For more details, see https://github.com/gardener/cc-utils/pull/497

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
Please check also the current `draft` release notes for this component. Before the release you can still update the pull requests that contain release notes, to adapt them to the new categories if needed.

**Release note**:

```improvement operator

```

### CODE REVIEW

**Review by rfranzke — DISMISSED** [review]
/lgtm

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ Write your release note:
 
 Format of block header: <category> <target_group>
 Possible values:
-- category:       improvement|noteworthy|action
-- target_group:   user|operator|developer
+- category:       breaking|feature|bugfix|doc|other
+- target_group:   user|operator|developer|dependency
 -->
-```improvement operator
+```bugfix operator
```
Comment (timebertt) [inline .github/pull_request_template.md]: Shouldn't the default rather be `other`?

```suggestion
```other operator
```

Comment (petersutter) [reply .github/pull_request_template.md]: I leave that up to you, I was also not 100% sure which one is the best default. If it is okay for the others, I will change it 👍

Comment (timebertt) [reply .github/pull_request_template.md]: Seems like, `other` is good :)

Comment (petersutter) [reply .github/pull_request_template.md]: done

**Review by rfranzke — APPROVED** [review]

**Review by timebertt — APPROVED** [review]
Thanks!
/lgtm

**Review by vpnachev — APPROVED** [review]
/lgtm

**Pull request merged**